Vagrantfile

```diff
@@ -3,11 +3,13 @@
 
 Vagrant.configure("2") do |config|
 
+  config.vagrant.plugins = [ "vagrant-vbguest" ]
+
   config.vm.define "centos-7" do |box|
     box.vm.box = "centos/7"
     box.vm.box_version = "1710.01"
     box.vm.hostname = "centos-7.local"
-    box.vm.network "private_network", ip: "192.168.48.101"
+    box.vm.network "private_network", ip: "192.168.58.101"
```

Comment:
The rationale for this and the following changes (c8a5e6230d8295db7cc48b6d984eeb8ade0ee408):

```
   Fix host-only network IPs to work with latest VirtualBox
    
    Virtualbox versions after 6.1.26 only allow host-only networks to be in the
    192.168.56.0/21 range. This behavior can be changed in
    /etc/vbox/networks.conf but would require local modifications for every user
    of this repository. Therefore give up and change the IP range to what
    new VirtualBox versions accept out of the box. Removal of the 192.168.48.0
    network from VirtualBox may be necessary when working with pre-existing VMs.
```